packages/postgres-database/src/simcore_postgres_database/utils_tags_sql.py

```diff
@@ -57,24 +54,42 @@ def get_tag_stmt(
     user_id: int,
     tag_id: int,
 ):
-    return sa.select(*_COLUMNS).select_from(
-        _join_user_to_given_tag(
-            access_condition=tags_access_rights.c.read.is_(True),
-            tag_id=tag_id,
-            user_id=user_id,
+    return (
+        sa.select(
+            *_TAG_COLUMNS,
+            # aggregation ensures MOST PERMISSIVE policy of access-rights
+            sa.func.bool_or(tags_access_rights.c.read).label("read"),
```

Comment:
So `bool_or` here is basically alternative to maximum :1:

Reply:
cannot use max in bools :-)